lib/bandit.ex

```diff
@@ -110,6 +113,7 @@ defmodule Bandit do
           max_header_length: pos_integer(),
           max_header_count: pos_integer(),
           max_requests: pos_integer(),
+          gc_every_n_requests: pos_integer(),
```

Comment:
Given this config only affects http1 keepalive requests, I would change the key name here to `gc_every_n_keepalive_requests` to avoid arresting more attention than necessary. 

As a developer, I can intuit why a gc config might be necessary for long lived keepalives, but I'd be somewhat suspicious of a config for any and all http1 reqs.

Reply:
It's already inside the http_1_options subgroup

Reply:
Updated.